### ISSUE TITLE
target xml node not inserted if not present.

### ISSUE THREAD

**Issue by thejeff77** [jasonruesch/xlifftranslate#2]
The code seems to translate everything correctly, however in the xliff document, nodes that didn't already have a target entry don't result in one being inserted with the translation.

```
      <trans-unit id="end_with_spaces">
        <source>  End</source>
        <target xml:lang="es" state="translated">  Fin</target>
        <note>End</note>
      </trans-unit>
      <trans-unit id="import_alert_title">
        <source>Import Files Status</source>
        <note>Import Files Status</note>
      </trans-unit>
```

Although the console shows the translation output:

es: Import Files Status => Importar estado de archivos

**Comment by thejeff77** [comment]
Fixed by PR #3

**Comment by thejeff77** [comment]
Not a bug. Target should be populated on export
